asconfig.json

```
@@ -0,0 +1,20 @@
+{
+  "targets": {
+    "debug": {
+      "binaryFile": "build/untouched.wasm",
+      "textFile": "build/untouched.wat",
+      "sourceMap": true,
+      "debug": true
+    },
+    "release": {
+      "binaryFile": "build/optimized.wasm",
+      "textFile": "build/optimized.wat",
+      "sourceMap": true,
+      "optimizeLevel": 3,
+      "shrinkLevel": 1,
+      "converge": false,
+      "noAssert": false
```

```suggestion
      "shrinkLevel": 0,
      "converge": false,
      "noAssert": true
```
if you required best perf

---

Thanks! @MaxGraey